client/src/hooks/subgraph/useEpochsStartEndTime.ts

```diff
@@ -45,7 +45,7 @@ export default function useEpochsStartEndTime(): UseQueryResult<
       request(subgraphAddress, GET_EPOCHS_START_END_TIME, {
         lastEpoch: currentEpoch,
       }),
-    queryKey: QUERY_KEYS.epochesEndTime(currentEpoch!),
+    queryKey: QUERY_KEYS.epochesEndTime(currentEpoch ?? -1),
```

Comment:
Why?

Reply:
QueryKey function invokes before `currentEpoch` is fetched so I declared `-1` as a default value because epoch `-1` doesn't exist.